app/views/needs/_notes.html.erb

```diff
@@ -2,6 +2,16 @@
   
   <%= form_with(model: [@need, @need.notes.build], local: true, class: "notes__form") do |form| %>
     <h3 class="notes__title">Updates</h3>
+    <div class="field">
+      <%= form.label :category, 'Choose a category', class: "field__label" %>
+      <% Note.categories.each.with_index do |nc, index| %>
```

Comment:
Rails has a form helper [`collection_radio_buttons`](https://apidock.com/rails/v4.0.2/ActionView/Helpers/FormOptionsHelper/collection_radio_buttons) that could simplify this and eliminate the need for the helpers above. But I couldn't get it to set the default option to general/note.

```ruby
      <%= form.collection_radio_buttons(:category, Note.categories, :last, :first) do |b| %>
        <div class="radio">
          <%= b.radio_button %>
          <%= b.label(class: 'radio__label') %>
        </div>
      <% end %>
```

No big deal either way - your code works and LGTM